src/rdkafka_conf.c

```diff
@@ -59,6 +59,7 @@ struct rd_kafka_property {
 	enum {
 		_RK_C_STR,
 		_RK_C_INT,
+                _RK_C_DBL,  /* Double */
```

Comment:
I guess this is better than specifying an integer number of microseconds.

Reply:
That would be a breaking change, changing linger.ms->linger.us, or worse; to add a new property linger.us.